types/reactstrap/lib/NavLink.d.ts

```diff
@@ -9,6 +9,7 @@ export interface NavLinkProps extends React.HTMLProps<HTMLAnchorElement> {
   cssModule?: CSSModule;
   onClick?: React.MouseEventHandler<any>;
   href?: string;
+  to?: string;
```

Comment:
I really hate to do this, but this isn't the correct way to do this as this isn't a reactstrap prop. See related discussion on #25329 that never got resolved.

I mentioned this in another comment chain on a different PR, but I've been meaning to put together a PR following the 

    export type RowProps<T> =  React.HTMLProps<HTMLElement> & T & {
        ...
    }
    declare class Row<T extends object = {}> extends React.Component<RowProps<T>> {}
    export default Row;

style syntax for all of the components, I just haven't had enough free time recently. Once #25921 gets merged, I'll try to put together a half hour or so to get that done since it'll cause a merge conflict.